subxt/src/rpc/rpc.rs

```diff
@@ -169,6 +169,25 @@ impl<T: Config> Rpc<T> {
         Ok(metadata)
     }
 
+    /// Execute a runtime API call.
+    pub async fn call(
+        &self,
+        function: String,
+        call_parameters: Option<&[u8]>,
```

Comment:
this assumes that the `call_parameters` is scale encoded right?

Reply:
Yes, we should provide the parameters scale-encoded concatenated and then apply `to_hex` internally.